app/assets/stylesheets/exercises/show.css.scss

```diff
@@ -62,6 +62,59 @@
   position: fixed;
   left: calc(33%);
 
+  #success, #fail {
+    display: none;
+    position: absolute;
+    top:0;
+    height: 4.6em;
+    width: 100%;
+    border: 0.2em solid $white;
+    padding-left: 1.5em;
+    padding-top:1em;
```

Comment:
one line for different paddings: `padding: 1em 0 0 1.5em`. Numbers start from the top and go clock wise.